src/OrbitGl/TrackContainer.cpp

```diff
@@ -384,6 +384,11 @@ void TrackContainer::UpdateVerticalScroll(float ratio) {
   SetVerticalScrollingOffset(new_scrolling_offset);
 }
 
+void TrackContainer::ScrollVertically(int delta) {
+  const int kPixelsPerDeltaInPanning = 25;
```

Comment:
kPixelsPerDeltaInPanning -> kPixelsPerDelta. The function is "ScrollVertically" yet the variable refers to panning.